Library/Homebrew/cmd/info.rb

```diff
@@ -7,8 +7,8 @@
 #:    To view formula history locally: `brew log -p <formula>`.
 #:
 #:  * `info` `--json=`<version> (`--all`|`--installed`|<formulae>):
-#:    Print a JSON representation of <formulae>. Currently the only accepted value
-#:    for <version> is `v1`.
+#:    Print a JSON representation of <formulae>.
+#:    Current options for <version>: `v1|pretty|lines`.
```

Comment:
I think this would have to include the version in its name somewhere. If we eventually add `--json=v2`, what does `pretty` return then?